Resources/Textures/SS220/Clothing/Mask/Blue-shield-mask.rsi/meta.json

```
@@ -0,0 +1,27 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Createed based on SS13 Paradise by MrALPAKA for ss220",
```

```suggestion
    "copyright": "Createed based on SS13 Paradise by MrALPAKA for ss220",
```
На чём based? Я конечно понимаю, база и всё такое, но если делается респрайт - всё равно должен быть указан коммит, с которого взята текстура. Либо пермалинк.